src/client/components/Sidebar/User.less

```diff
@@ -28,9 +28,13 @@
 }
 
 .User__follow {
-  display: flex;
+  width: 40%;
```

Comment:
why add width... to follow button?

Reply:
sorry can you delet those change its for another issu !!

Reply:
not too sure what you mean? you can edit your own PR :)

Reply:
i'l making another pr but it was mixed with this one i want to delet those commit !!